File: Inshortsnewsapp/app/src/main/java/com/example/inshortsnewsapp/ResponseModel.java
package com.example.inshortsnewsapp;

import java.util.List;
import com.google.gson.annotations.SerializedName;
import java.io.Serializable;

public class ResponseModel implements Serializable {

	@SerializedName("category")
	private String category;

	@SerializedName("data")
	private List<DataModel> data;

	@SerializedName("success")
	private boolean success;

	public String getCategory(){
		return category;
	}

	public List<DataModel> getData(){
		return data;
	}

	public boolean isSuccess(){
		return success;
	}

	@Override
 	public String toString(){
		return 
			"ResponseModel{" + 
			"category = '" + category + '\'' + 
			",data = '" + data + '\'' + 
			",success = '" + success + '\'' + 
			"}";
		}
}

File: KotlinScopingFunctionsIW/app/src/main/java/com/example/kotlinscopingfunctionsiw/ResponseModel.kt
package com.example.kotlinscopingfunctionsiw

import com.google.gson.annotations.SerializedName

data class ResponseModel(

	@field:SerializedName("data")
	val data: DataModel? = null,

	@field:SerializedName("support")
	val support: SupportModel? = null
)